<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Test Project Automation Suite" duration-ms="69308" started-at="2019-05-07T20:00:22Z" finished-at="2019-05-07T20:01:32Z">
    <groups>
    </groups>
    <test name="Functional Automation Tests" duration-ms="69308" started-at="2019-05-07T20:00:22Z" finished-at="2019-05-07T20:01:32Z">
      <class name="com.qa.testcases.ContactPageTest">
        <test-method status="PASS" signature="begin()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="begin" is-config="true" duration-ms="217" started-at="2019-05-07T13:00:22Z" finished-at="2019-05-07T13:00:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- begin -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="setUp" is-config="true" duration-ms="13339" started-at="2019-05-07T13:00:22Z" finished-at="2019-05-07T13:00:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyContactPageHeadingTest()[pri:1, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="verifyContactPageHeadingTest" duration-ms="93" started-at="2019-05-07T13:00:36Z" finished-at="2019-05-07T13:00:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyContactPageHeadingTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="tearDown" is-config="true" duration-ms="579" started-at="2019-05-07T13:00:36Z" finished-at="2019-05-07T13:00:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="setUp" is-config="true" duration-ms="12313" started-at="2019-05-07T13:00:36Z" finished-at="2019-05-07T13:00:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyNewContactPageTest()[pri:2, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="verifyNewContactPageTest" duration-ms="139" started-at="2019-05-07T13:00:49Z" finished-at="2019-05-07T13:00:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNewContactPageTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="tearDown" is-config="true" duration-ms="598" started-at="2019-05-07T13:00:49Z" finished-at="2019-05-07T13:00:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="setUp" is-config="true" duration-ms="11714" started-at="2019-05-07T13:00:50Z" finished-at="2019-05-07T13:01:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="createNewContactTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:3, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="createNewContactTest" duration-ms="7860" started-at="2019-05-07T13:01:02Z" data-provider="getNewContactData" finished-at="2019-05-07T13:01:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Rehan]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Sharaf]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[11]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[November]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[1988]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.qa.testcases.ContactPageTest.createNewContactTest(ContactPageTest.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewContactTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="tearDown" is-config="true" duration-ms="614" started-at="2019-05-07T13:01:10Z" finished-at="2019-05-07T13:01:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="setUp" is-config="true" duration-ms="11962" started-at="2019-05-07T13:01:11Z" finished-at="2019-05-07T13:01:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="createNewContactTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:3, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="createNewContactTest" duration-ms="8112" started-at="2019-05-07T13:01:23Z" data-provider="getNewContactData" finished-at="2019-05-07T13:01:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Adnan]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Sharaf]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[December]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[1974]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.qa.testcases.ContactPageTest.createNewContactTest(ContactPageTest.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewContactTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="tearDown" is-config="true" duration-ms="640" started-at="2019-05-07T13:01:31Z" finished-at="2019-05-07T13:01:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setExtent()[pri:0, instance:com.qa.testcases.ContactPageTest@de3a06f]" name="setExtent" is-config="true" duration-ms="304" started-at="2019-05-07T13:01:32Z" finished-at="2019-05-07T13:01:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
      </class> <!-- com.qa.testcases.ContactPageTest -->
    </test> <!-- Functional Automation Tests -->
  </suite> <!-- Test Project Automation Suite -->
</testng-results>
